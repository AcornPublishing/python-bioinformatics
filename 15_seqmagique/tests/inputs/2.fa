>SEQ0
GGGCATTGCTAACGATACGTACTCTATACGCCTAATGAATAAGGAATTTTGACTAGATAGAGTCAGACTCGAGATATCC
>SEQ1
TCCTAAATACTATCCTGCTGTGCAAGAGCCCCCACTCCCCACTGAGAATCTCCTGTGCG
>SEQ2
AATTTTTAGTCCTCCATATTTTCGCGGTGCTCAACCGAATCAAGTTCGTATCACGACAC
>SEQ3
GCAAGGTGGTCTGTCTTGATGTAGTTCTGGCAAGGTTTATGTATCGTGTTAACCTACTTTCTCAAGGCTTCCAA
>SEQ4
GAGATCGACCTACCTCGAGAGGAAGCTCCGCAAAATATTTAGGGAAAGG
