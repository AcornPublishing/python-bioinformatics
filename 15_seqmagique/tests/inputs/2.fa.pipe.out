| name                |   min_len |   max_len |   avg_len |   num_seqs |
|:--------------------|----------:|----------:|----------:|-----------:|
| ./tests/inputs/2.fa |        49 |        79 |     64.00 |          5 |
